docs/metrics.md

```diff
@@ -3,15 +3,15 @@ This document aims to help users that are not familiar with metrics exposed by t
 All metrics documented here are auto-generated by the utility tool `tools/metricsdocs` and reflects exactly what is being exposed.
 
 ## SSP Operator Metrics List
-### kubevirt_ssp_common_templates_restored_total
-The total number of common templates restored by the operator back to their original state. Type: Counter.
-### kubevirt_ssp_num_of_operator_reconciling_properly
+### kubevirt_ssp_common_templates_restored_increase
+The total number of common templates restored by the operator back to their original state. Type: Gauge.
+### kubevirt_ssp_operator_reconcile_succeeded_aggregated
 The total number of ssp-operator pods reconciling with no errors. Type: Gauge.
-### kubevirt_ssp_operator_up_total
+### kubevirt_ssp_operator_up
 The total number of running ssp-operator pods. Type: Gauge.
-### kubevirt_ssp_rejected_vms_total
-The total number of vms rejected by virt-template-validator. Type: Counter.
-### kubevirt_ssp_template_validator_up_total
+### kubevirt_ssp_template_validator_rejected_increase
+The total number of vms rejected by virt-template-validator. Type: Gauge.
+### kubevirt_ssp_template_validator_up
```

Comment:
@assafad please add the missing recording rule to this file as well. Like: kubevirt_ssp_template_validator_rejected_total

Reply:
The ones that are not collected to `metrics.md` are metrics (not recording rules), which are scattered across the repo.  They were not collected before this PR. Collecting them would require some changes in the metrics docs automation, and thus will complicate the review. I would be happy to submit a follow-up PR for that, but please note that it will no longer be needed after the monitoring refactoring, so we can also wait with it.

Reply:
Please add them manually for now. We did that in other repos as well.

Reply:
A follow up sounds fine to me.

Reply:
Updated in this PR.